src/__mocks__/react-intl.js

```diff
@@ -0,0 +1 @@
+export * from '../../scripts/jest/mocks/react-intl-mocks';
```

Comment:
exclude mocks folder from babel compile, within package.json scripts, similar to tests